assignments/this.js

```diff
@@ -1,26 +1,57 @@
 /* The for principles of "this";
 * in your own words. explain the four principle for the "this" keyword below.
 *
-* 1. 
-* 2. 
-* 3. 
-* 4. 
+* 1. default binding = automatically binds to the window
+* 2. implicit binding = binds to the left of the dot
+* 3. explicit binding = attaches to the function call
+* 4. new binding = a new object is created and attached to the this
 *
 * write out a code example of each explanation above
 */
 
 // Principle 1
 
-// code example for Window Binding
+const myGreatestFunction = function () {
```

Comment:
I would try these again using this as your guide: https://tylermcginnis.com/this-keyword-call-apply-bind-javascript/